docs/source/start-screening.rst

```diff
@@ -0,0 +1,6 @@
+################################
```

Comment:
Would QA/Review work better here?

If screening is the appropriate generic term then yeah we should use it but we should verify. And at the least mention any synonyms for search.

Reply:
The window is called "Screening Editor". 
![image](https://github.com/opendcs/opendcs/assets/104451600/2ff1ecd1-4b3e-4e83-868a-bcd089b66b44)
I think this could be a greater discussion, but to me screening is accurate, since it is near-real-time "screening" vs a more rigorous quality assurance/control.  I will look to add some synonyms and include some background in this section.